helpers/test/test_hmarkdown.py

```diff
@@ -1037,3 +1037,256 @@ def test8(self) -> None:
         expected = "\n".join(expected)
         actual = hio.from_file(write_file)
         self.assertEqual(actual, expected)
+
+
+def _get_markdown_example7() -> str:
+    content = r"""
+        ```python
+        def hello_world():
+            print("Hello, World!")
+        ```
+        """
+    content = hprint.dedent(content)
+    return content
+
+
+def _get_markdown_example8() -> str:
```

Comment:
If it's used in a single place, it's not worth introducing a new symbol but best to inline